python/effects/stop_the_light.py

```diff
@@ -26,7 +26,8 @@
 BLACK = (0, 0, 0)
 
 class StopTheLight(Effect):
-    def __init__(self, strip_bottom=0, strip_top=15):
+    def __init__(self, layout, n_pixels, strip_bottom=5, strip_top=20):
+        Effect.__init__(self, layout, n_pixels)
```

Comment:
Just a really common operation avoiding extra len() functions. But yeah, we could just take layout and do len on it in Effect#__init__